aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: kubernetes-dashboard
-version: 4.0.2
+version: 4.0.3
```

Comment:
Could you bumb minor version instead of patch version? According to semver, this is a new feature so we need to bump!